src/app/layout.tsx

```diff
@@ -18,7 +18,7 @@ export default function RootLayout({
   children: React.ReactNode;
 }>) {
   return (
-    <html lang="en">
+    <html lang="en" suppressHydrationWarning={true}>
```

Comment:
I don't thik that suppress warnings is a good idea. Could You try to solve the hydration wornings?

Reply:
The next-themes causes those warnings, and i can`t fix them directly.